honeycomb-core/src/cmap/components/unused.rs

```diff
@@ -22,6 +22,11 @@ impl UnusedDarts {
         self.0.extend((0..len).map(|_| TVar::new(false)));
     }
 
+    /// Extend internal storage capacity using the passed value for new items
+    pub fn extend_from_val(&mut self, len: usize, val: bool) {
```

Comment:
```suggestion
    pub fn extend_with(&mut self, len: usize, val: bool) {
```
Not sure about the semantic you want to follow, but this looks closer to what people are doing externally.

Reply:
yeah, I have no strong opinion on the naming, doing what others do is probably best

I'll need to properly update everything in a single commit if I want to rebase this tho